server/seed/generateItems.js

```diff
@@ -0,0 +1,33 @@
+const mongoose = require('mongoose');
+
+const {
+  Types: { ObjectId }
+} = mongoose;
+
+const generateItems = userId => itemsCount => {
+  const items = [];
+
+  for (let i = 0; i < itemsCount; i += 1) {
```

Comment:
`i++`

Reply:
![Zrzut ekranu 2019-09-9 o 11 34 19](https://user-images.githubusercontent.com/27632432/64520137-c8d91080-d2f5-11e9-9be8-6c9f0b992594.png)

Why` i++` ??
Till now we have used `i += 1` across the whole project

Reply:
BTW eslint disallows
![Zrzut ekranu 2019-09-9 o 11 40 01](https://user-images.githubusercontent.com/27632432/64520519-a4316880-d2f6-11e9-8054-97d7959564dc.png)
 usage of i++

Reply:
please check rule: { "allowForLoopAfterthoughts": true } 
https://eslint.org/docs/rules/no-plusplus

In loops there is no problem with using it